apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -25,12 +26,20 @@ import Instructions from './Instructions';
 import CollapserIcon from './CollapserIcon';
 import HeightResizer from './HeightResizer';
 import msg from '@cdo/locale';
+import experiments from '@cdo/apps/util/experiments';
+import { ViewType } from '@cdo/apps/code-studio/viewAsRedux';
 
 const HEADER_HEIGHT = styleConstants['workspace-headers-height'];
 const RESIZER_HEIGHT = styleConstants['resize-bar-width'];
 
 const MIN_HEIGHT = RESIZER_HEIGHT + 60;
 
+const tab = {
+  instructions: 'instructions',
+  resources: 'resources',
+  comments: 'comments'
+};
```

Comment:
I think our style is more like this:
```
const TabType = {
  INSTRUCTIONS: 'instructions',
  RESOURCES: 'resources',
  COMMENTS: 'comments'
};
```

Example: https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/templates/sectionProgress/sectionProgressRedux.js#L88